x509-cert/src/builder.rs

```diff
@@ -161,20 +166,18 @@ impl Profile {
         // Build Key Usage extension
         match self {
             Profile::Root | Profile::SubCA { .. } => {
-                extensions.push(
-                    KeyUsage(
-                        KeyUsages::DigitalSignature | KeyUsages::KeyCertSign | KeyUsages::CRLSign,
-                    )
-                    .to_extension(tbs)?,
-                );
+                extensions
+                    .push(KeyUsage(KeyUsages::KeyCertSign | KeyUsages::CRLSign).to_extension(tbs)?);
```

Comment:
`KeyUsages::DigitalSignature` is getting dropped, I don't remember why I added it in the first place.

Reply:
@baloo it is reasoned in the commit message. Please review by the commit.